.gitignore

```diff
@@ -25,3 +25,4 @@ e2e/test-report/
 e2e/test-results/
 **/.cache/
 docs/
+__screenshots__
```

Comment:
added this to gitignore because now if test fails there is a screenshot folder so you can see how page looked like at the time of failure
for example, if your test had canvas appended to the document, you can see how canvas looked like when it failed the test